src/game/main.cpp

```diff
@@ -269,6 +276,43 @@ void Create_Window()
 #endif
 }
 
+void Create_Window_SDL2()
+{
+#ifdef BUILD_WITH_SDL2
+    bool is_windowed = g_gameIsWindowed;
+    SDL_Window *window = NULL;
+
+    if (SDL_Init(SDL_INIT_VIDEO) < 0) {
+        captainslog_error("SDL could not initialize!");
+        return;
```

Comment:
Shouldn't this function return false and then trickle up to terminate the game process? Or is the process expected to continue after this error?